tests/e2e/test_spyre_basic.py

```diff
@@ -1,5 +1,10 @@
 """Verification of vLLM output by comparing with HF
 
+If errors occur, these can be analyzed/debugged by setting
+`DISABLE_ASSERTS = True` in spyre_util.py and by rerunning the
+test using `pytest --capture=no tests/spyre/test_spyre_basic.py`
+After debugging, `DISABLE_ASSERTS` should be reset to `False`.
```

Comment:
This line is in multiple tests, wondering if w can keep it someplace central

Reply:
I guess? Makes more sense than having the same content in different test files. Open to other ideas too